src/components/Books.js

```diff
@@ -0,0 +1,25 @@
+import '../assets/Books.css';
+
+const Books = () => (
+  <div className="books">
+    <ul className="book">
+      <li>
+        <h2>Book 1</h2>
+        <button type="button">Remove</button>
+      </li>
+      <li>
+        <h2>Book 2</h2>
+        <button type="button">Remove</button>
+      </li>
+    </ul>
+    <hr />
+    <h2 className="add-book">Add new Book</h2>
+    <form className="form">
+      <input type="text" placeholder="Book title" />
+      <input type="select" placeholder="Category" />
+      <input type="submit" className="submit" value="Add Book" />
+    </form>
+  </div>
+);
```

Comment:
- [ ] In this component right now you are just displaying HTML code it doesn't really function as intended. the books component should render the `Book` and `CreateBook` components.
- the `Book` component takes the book data as a prop and renders the title and the author and has a button for removing books
- the `CreateBook` component renders a form with inputs for the title and the author of the book and has a button to submit
Both the button for removing books and for submitting the form should be placeholder buttons right now they don't have to do anything.

Reply:
Thank you :+1:  I implemented the necessary changes